docs/extensions/sqlite_scanner.md

```diff
@@ -129,6 +131,6 @@ If you want to run the `sqlite_scan` procedure more than once in the same DuckDB
 CALL sqlite_attach('sakila.db', overwrite=true);
 ```
 
-## Extra Information
+## Source Code
```

Comment:
```suggestion
## GitHub Repository
```
? Given it's not only about the source, but also issue tracking / PRs and what not